torch/csrc/jit/passes/quantization.cpp

```diff
@@ -1277,26 +1328,49 @@ void insertQuantDeQuantCall(
   } else {
     quantize_func = "quantize_per_tensor";
   }
-  Node* quant = g->create(at::Symbol::aten(quantize_func), inputs);
-  quant->output()->setDebugName(v->debugName() + ".quant");
-  g->insertNode(quant);
+  Node *quant, *choose_qparams;
+  if (is_dynamic && !isWeightOfConvOrLinear(v)) {
+    std::string choose_qparams_func = "_choose_qparams_per_tensor";
+    auto reduce_range = g->insertConstant(false);
+    choose_qparams = g->create(at::Symbol::aten(choose_qparams_func), v, 2);
```

Comment:
what is `2`?

Reply:
looks like this is number of outputs. can you add a comment?
also can you put all inputs together?
like 
`g->create(at::Symbol::aten(choose_qparams_func), {v, reduce_range}, 2);`